src/presentation/controllers/ingest-controller.ts

```diff
@@ -0,0 +1,24 @@
+import { BaseController, PostRequest, RestController } from './base-controller'
+import { HttpErrorHandler } from '../interfaces/http-error-handler'
+import { Exception } from '../../model/exceptions/exception'
+import { Request, Response } from 'express'
+import { IngestService } from '../../business-logic/services/interfaces/ingest-service'
+
+@RestController('/ingest')
```

Comment:
This is a quite interesting scenario.
To follow REST standards we should use plural forms, but plural of `ingest` doesn't make any sense.
Maybe this shows that the right controller might be the `RundownController` with a URL of `rundowns/{rundownId}/reloadData`?

Reply:
I agree this is interesting. I wanted a new controller since the ingest is not directly a manipulation of a rundown. Triggering this action does not directly signify that we alter the rundown, just that trigger the ingest to reload the data. For that reason i kept it seperate. I see the point about `ingest` but i don't have a better suggestion for now...

Reply:
I'm also in favour of having the `IngestController`.
I'm not sure if it is better, but we could call it `ingestOperations` or something similar?
Or we could just keep it as is. Just not the biggest fan of not following REST conventions in a REST endpoint.

Reply:
Reloading the data for a rundown is an action on the rundown. The frontend(s) (probably) shouldn't be aware about ingest data. From the frontend(s) perspective, it/they are still doing an operation on a rundown. When we get to actually receiving data from our ingest gateway, then an ingest controller, which the ingest gateway interacts with, will be appropriate.

Reply:
I have changed the URL to be `ingest-operations` instead, let me know if this is not what you wanted. 
Also, i see your point @KvelaGorrrrnio, but is triggering the data reload from the gateway really an action on the rundown itself in the same way as activating it is? If we'd rather have it inside the `RundownController`, just let me know and i will change it!